### PR TITLE
Cypress e2e PF6 fixes

### PR DESCRIPTION
**Pull request by FedeAlonso**
https://issues.redhat.com/browse/RHOAIENG-16583 

## Description
This PR Contains the fixes needed in order to make the e2e Cypress run properly.

## How Has This Been Tested?
...job/cypress/job/dashboard-tests/120: 
image
**The Test that it's failing is due to RHOAIENG-16609**

## Test Impact
Just the e2e has been reviewed

## Request review criteria:
I'm open to review and critics. Please be gentle.

Self checklist (all need to be checked):a
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Review by manosnoam — COMMENTED** [review]
/lgtm

File: frontend/src/__tests__/cypress/cypress/tests/e2e/settings/servingRuntimes/testMultiModelServingRuntimeCreation.cy.ts [hunk 1]
```diff
@@ -68,8 +68,8 @@ describe('Verify Admins Can Import and Delete a Custom Multi-Model Serving Runti
       .find()
       .within(() => {
         servingRuntimes.findEditModel().click();
-        servingRuntimes.findDeleteModel().click();
       });
+    servingRuntimes.findDeleteModel().click();
```
Comment (antowaddle) [inline frontend/src/__tests__/cypress/cypress/tests/e2e/settings/servingRuntimes/testMultiModelServingRuntimeCreation.cy.ts:72]: Please also make the same change to the _testSingleServingRuntimeCreation.cy_ test which is currently failing.

**Comment by antowaddle** [comment]
Another test is also failing - _clusterStorage.cy.ts_ Can that also be checked please? 

image

File: frontend/src/__tests__/cypress/cypress/tests/e2e/settings/servingRuntimes/testMultiModelServingRuntimeCreation.cy.ts (hunk 1 above)
Comment (FedeAlonso) [reply frontend/src/__tests__/cypress/cypress/tests/e2e/settings/servingRuntimes/testMultiModelServingRuntimeCreation.cy.ts:72]: Fixed in [c7a3daf](https://github.com/opendatahub-io/odh-dashboard/pull/3549/commits/c7a3daffb15eff5383f55f7a7670f3e319b35669)

**Comment by FedeAlonso** [comment]
> Another test is also failing - _clusterStorage.cy.ts_ Can that also be checked please?
> 
> image

This test is failing because there's a bug: RHOAIENG-16609

**Comment by manosnoam** [comment]
> This test is failing because there's a bug: [RHOAIENG-16609](https://issues.redhat.com//browse/RHOAIENG-16609)

Thanks for reporting this bug that was found via Cypress E2E!
Until we add a common annotation for "known bugs", please add to the relevant E2E test that failed due to this bug a message like cy.STEP("Product Bug: [RHOAIENG-16609](https://issues.redhat.com/browse/RHOAIENG-16609)")

**Review by manosnoam — COMMENTED** [review]
/lgtm

**Review by antowaddle — APPROVED** [review]
/lgtm

**Pull request merged**